components/autofill/Cargo.toml

```diff
@@ -23,7 +23,7 @@ sync15 = { path = "../sync15" }
 sync15-traits = {path = "../support/sync15-traits"}
 thiserror = "1.0"
 types = { path = "../support/types" }
-uniffi = "^0.10"
+uniffi = "^0.11"
```

Comment:
doh! Not sure what went wrong here, but I didn't mean to upgrade uniffi, and nothing here depends on it.